Jenkinsfile.jssdk

```diff
@@ -0,0 +1,44 @@
+#!groovy
+ansiColor('xterm') {
+  timestamps {
+    timeout(60) {
+      node('NODE_JS_BUILDER') {
+        stage('Install') {
+          withCredentials([
+            string(credentialsId: 'NPM_TOKEN', variable: 'NPM_TOKEN')
+          ]) {
+            sh 'echo \'//registry.npmjs.org/:_authToken=${NPM_TOKEN}\' > $HOME/.npmrc'
+            sh '''#!/bin/bash -ex
+            source ~/.nvm/nvm.sh
+            nvm install 7
+            nvm use v7
+            npm install
+            rm -f $HOME/.npmrc
+            '''
+          }
+        }
+
+        stage('Upgrade') {
+          echo "Upgrading @ciscospark packages"
+          sh '''#!/bin/bash -ex
+          source ~/.nvm/nvm.sh
+          nvm use v7
+          npm run upgradespark
+          '''
+        }
+
+        stage('Branching') {
+          echo "Creating Branch"
+          sh '''#!/bin/bash -ex
+          version=`npm view ciscospark version`
```

Comment:
use $() instead of backticks. same effect, but nestable, so it's a good habit to get into.